docs/docs/behaviors/hold-tap.md

```diff
@@ -50,13 +50,19 @@ In QMK, unlike ZMK, this functionality is enabled by default, and you turn it of
 
 #### `retro-tap`
 
-If retro tap is enabled, the tap behavior is triggered when releasing the hold-tap key if no other key was pressed in the meantime.
+If retro tap is enabled, the tap behavior is triggered when releasing the hold-tap key if no other key was pressed in the meantime. The hold key does not activate until another key is pressed, meaning that it cannot be used for mouse events like Shift Click to select from your cursor position to mouse position.
 
-For example, if you press `&mt LEFT_SHIFT A` and then release it without pressing another key, it will output `a`.
+For example, if you press `&sh LEFT_SHIFT Z` and then release it without pressing another key, it will output `z`.
 
 ```
-&mt {
-	retro-tap;
+sh: shift {
```

Comment:
this needs tobe within `/ { behaviors { ` (unlike the old example, which extended an existing behavior instead of defining a new one)